.github/workflows/test-go.yml

```diff
@@ -115,6 +122,11 @@ jobs:
           #
           ENTERPRISE=${{ inputs.enterprise }} source .github/scripts/generate-test-package-lists.sh
 
+          # If data race detection is enabled, export a relevant environment variable for use in tests
+          if [ "${{ inputs.extra-flags }}" == "-race" ]; then
```

Comment:
Don't you need to check for "contains" rather than equality?

Reply:
Good spot, thanks!